python/plugins/processing/algs/qgis/ServiceAreaFromPoint.py

```diff
@@ -137,8 +138,11 @@ def defineCharacteristics(self):
             p.isAdvanced = True
             self.addParameter(p)
 
-        self.addOutput(OutputVector(self.OUTPUT_LAYER,
-                                    self.tr('Service area'),
+        self.addOutput(OutputVector(self.OUTPUT_POINTS,
```

Comment:
Ok. It'd be a nice enhancement we could add at a future stage anyway. +1 to merge!